File: src/control/load/databaseReader.py
from src.model.game.Game import Game
from src.model.player.PlayerContainer import PlayerContainer
from src.model.player.Player import Player
from src.model.card.Card import Card
from src.model.collection.Collection import Collection

from src.control.load.Effects import getDirectDamage
from src.control.load.Effects import getSitsNTurns

def getGame(startInfo):
	uids = startInfo[0]
	dids = startInfo[1]
	#TODO GET PLAYER DECK INFO FROM DATABASE
	cardNames = ['farts','fresh','persist']
	players = []
	for i in xrange(0,2):
		player = Player(uids[i])
		cards = []
		name = cardNames[2]
		effect = getSitsNTurns('water',3)
		cards.append(Card(name,effect))
		for j in xrange(0,39):
			name = cardNames[i]
			effect = getDirectDamage('fire',4)
			cards.append(Card(name,effect))
		playerCollection = Collection(cards)
		players.append(PlayerContainer(player,playerCollection))

	return Game(players[0],players[1])

File: src/jhost/control/DatabaseReader.java
package control;

import java.util.ArrayList;

import control.CardLoader;
import model.ElementType;
import model.Game;
import model.PlayerContainer;
import model.Player;
import model.Deck;
import model.Card;
import model.Effect;

public class DatabaseReader{
	public static Game getGame(int p1Uid, int p1Did, int p2Uid, int p2Did){
		PlayerContainer[] pcs = new PlayerContainer[2];
		for(int i=0;i<2;i++){
			Player player = new Player(1,"Quin");
			ArrayList<Card> cards = new ArrayList<Card>(10);
			int[] loadedDeck = {1,1,1,1,1,1,1,1,1,1};
			for(int cardId : loadedDeck){
				cards.add(CardLoader.getDirectDamage("Fire Blast",ElementType.fire,5));
			}
			Deck deck = new Deck(cards);
			pcs[i] = new PlayerContainer(player,deck);
		}
		return new Game(pcs[0],pcs[1]);
	}
}
